src/components/studies/app-design/GeneralContactAndSupportSection.tsx

```diff
@@ -0,0 +1,228 @@
+import React from 'react'
+import { makeStyles } from '@material-ui/core'
+import Subsection from './Subsection'
+import { FormControl, Box, FormHelperText } from '@material-ui/core'
+import { StudyAppDesign, Contact } from '../../../types/types'
+import { AppDesignUpdateTypes } from './AppDesign'
+import clsx from 'clsx'
+import { isInvalidPhone, isValidEmail } from '../../../helpers/utility'
+import { makePhone } from '../../../helpers/utility'
+import FormGroupWrapper from './FormGroupWrapper'
+import TextInputWrapper from './TextInputWrapper'
+import { ContactType } from './AppDesign'
+
+const useStyles = makeStyles(theme => ({
+  firstFormElement: {
```

Comment:
It's fine as it is, but I think when you have a situation where you have a single rule for a single instance it's fine to just have a style inline attribute